alerts.tf

```diff
@@ -183,15 +165,17 @@ resource "newrelic_nrql_alert_condition" "error_rate_4xx" {
   nrql {
     query = <<-EOF
         SELECT 100 * (
-          filter(count(*), WHERE ${var.response_status_variable_name} LIKE '4%') / (count(*) + 1e-10)
+          filter(count(*), WHERE ${var.response_status_variable_name} LIKE '${each.value.nrql_value}') / (count(*) + 1e-10)
         ) as Percentage
         FROM Transaction
         WHERE appName = '${var.newrelic_fully_qualified_app_name}'
-        ${var.alert_error_rate_4xx_conditions}
+        ${coalesce(each.value.extra_conditions, " ")}
```

Comment:
Can't it be an empty string?

Reply:
Unfortunately empty strings does not work with `coalesce` 😞